docs/projects-to-work-on.md

```diff
@@ -0,0 +1,428 @@
+  and projects implemented with it!
+
+### Miscellaneous
+
+- **[Ressource-Embedder](https://gitlab.com/stcarrez/resource-embedder)** is a
+  resource embedder for Ada, Go and C programs. It allows the user to load data
+  as part of the executable! You can help by utilising and promoting it.
+
+## Operating Systems
+
+The Ada ecosystem sports some of the most interesting kernels/OSes out
+there. Operating System development tends to be labeled as an advanced
+topic. However, if you are learning, thanks to Ada's clear syntax, it becomes
+much easier to understand them, use them and even contribute to an OS. Here is a
+list of kernels/OSes written in pure Ada/SPARK that may greatly benefit from the
+community:
+
+- **[Ironclad](https://ironclad.cx/)** is a fairly new kernel that strives to be
+  POSIX compatible. It partially written in SPARK, making it much more robust
+  and safe by default. It aims to support x86, ARM and RISC-V architectures. You
+  can contribute to it or you can help grow the
+  [Gloire](https://github.com/streaksu/Gloire) OS distribution, which uses
+  Ironclad as its kernel.
+- **[HiRTOS](https://github.com/jgrivera67/HiRTOS)** is a high integrity RTOS
+  kernel as well as a separation kernel written in SPARK. Go and give it a try!
+  One can even install it using Alire! It currently targets the ARM Cortex-R52
+  CPU. One could try and port it to a different CPU.
+- **[Muen](https://muen.codelabs.ch/)** is a separation kernel fully written in
+  SPARK. It has been certified by some large military bodies for their internal
+  use. It also sports some drivers fully written in SPARK!
+- **[M2OS](https://m2os.unican.es/)** is a small RTOS that was created from the
+  same people behind the Ada OS [Marte](https://marte.unican.es/).
+
+There are some well known Operating Systems that allow users to run Ada programs
+in them directly even if they are written in a different language. You may want
+to use them, improve the Ada support they have or write examples or blog posts
+about their use:
+
+- **[RTEMS](https://www.rtems.org)** is a POSIX RTOS that is widely used in
+  the aerospace industry. It has official Ada support. However, its use is not
+  clearly documented nor widely known about; this could be fixed.
+
+## Embedded Programming
+
+- **[Ada Drivers Library](https://github.com/AdaCore/Ada_Drivers_Library)** is a
+  collection of drivers for microcontollers and embedded boards. You can help by
+  utilising it, testing the built-in drivers and specially by helping improve
+  drivers and boards that are not yet present.
+
+- **Create more HALs:** HAL (Hardware Abstraction Layer) is a piece of code that
+  abstracts the device away from the code. This is what generally drivers do. By
+  implementing a HAL the low level aspects of the hardware are not needed in
+  order to program the device, which makes the experience of embedded
+  programming a lot more pleasant and readable. Ada has quite a few HALs (see
```

Comment:
```suggestion
  programming a lot more pleasant and comprehensible. Ada has quite a few HALs (see
```
I think "experience of programming" matches better to "comprehensible" than to "readable" (that could be said to the source code, not the experience).